### PR TITLE
docs: change gen_vimdoc indentation level

### PR DESCRIPTION
**Pull request by ii14**

### CODE REVIEW

**Review by dundargoc — APPROVED** [review]
I like it. The nested sections isn't squished as much and have more room to breathe.

**Comment by clason** [comment]
Note: if we do this, we should follow standard refactor procedure:
1. separate commit for change to script and change to generated files
2. add the latter to [.git-blame-ignore-revs](https://github.com/neovim/neovim/blob/master/.git-blame-ignore-revs).

**Comment by clason** [comment]
Also, it would be good to make the indentation consistent; it (currently and still with this PR) differs widely between sections even in the same file.

**Review by clason — APPROVED** [review]
That does look much better, thank you!

**Comment by ii14** [comment]
Some of the manually written parts were using hard tabs, which was again inconsistent with the generated parts. I've added `expandtab` to modeline.

File: scripts/gen_vimdoc.py [hunk 1]
```diff
@@ -1114,7 +1118,7 @@ def main(config, args):
             docs += '\n\n\n'
 
         docs = docs.rstrip() + '\n\n'
-        docs += ' vim:tw=78:ts=8:ft=help:norl:\n'
+        docs += ' vim:tw=78:ts=8:sw=4:sts=4:et:ft=help:norl:\n'
```
Comment (clason) [inline scripts/gen_vimdoc.py:1121]: if `indentation` is a variable now, might as well use that instead of hardcoding?

**Pull request merged**